set.mm

```diff
@@ -645698,6 +645854,20 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
       USUTRACUOSTT $.
   $}
 
+  $( ~ dvdssqlem generalized to positive integer exponents.  (Contributed by
+     SN, 20-Aug-2024.) $)
+  dvdsexpnn $p |- ( ( A e. NN /\ B e. NN /\ N e. NN ) ->
+                                    ( A || B <-> ( A ^ N ) || ( B ^ N ) ) ) $=
```

Comment:
True for any $A, B \in \mathbb{Z}$.

Reply:
(from my remark on `fltdvdsabdvdsc` below, this generalization will be left for another pr)